docusaurus/docs/deployment.md

```diff
@@ -324,7 +324,7 @@ additional modification:
   "scripts": {
     "predeploy": "npm run build",
 -   "deploy": "gh-pages -d build",
-+   "deploy": "gh-pages -b master -d build",
++   "deploy": "gh-pages -b main -d build",
```

Comment:
👍  Nice update, however, if we start changing this, I'd recommend that we do so consistently across the documentation & repository (which is currently still using `master`).

I've also noticed `master` is still reference on line 321: https://github.com/facebook/create-react-app/pull/9890/files#diff-c15f58740e5e395bec4ab999e4919181522ca758932a463ff27977aecb648a76R321